### PR TITLE
net: wifi_shell: Display TWT capability of AP in scan results

### PR DESCRIPTION
**Pull request by rado17**
Include TWT capability in scan results.

### CODE REVIEW

**Comment by MaureenHelm** [comment]
@rlubos can you take a look please?

File: subsys/net/l2/wifi/wifi_shell.c [hunk 1]
```diff
@@ -87,18 +87,19 @@ static void handle_wifi_scan_result(struct net_mgmt_event_callback *cb)
...
+	print(context.sh, SHELL_NORMAL, "%-4d | %-32s %-5u | %-4u (%-6s) | %-4d | %-15s | %s | %s\n",
 	      scan_result, entry->ssid, entry->ssid_length, entry->channel,
 	      wifi_band_txt(entry->band),
 	      entry->rssi,
 	      wifi_security_txt(entry->security),
 	      ((entry->mac_length) ?
 		      net_sprint_ll_addr_buf(entry->mac, WIFI_MAC_ADDR_LEN, mac_string_buf,
-					     sizeof(mac_string_buf)) : ""));
+					     sizeof(mac_string_buf)) : ""),
+		 entry->twt_support ? "Enabled":"Disabled");
```
Comment (rlubos) [inline subsys/net/l2/wifi/wifi_shell.c:102]: The alignment seems off. Also, need spaces around `:`

**Review by rlubos — COMMENTED** [review]
Please address the compliance check warnings.

**Comment by rado17** [comment]
Instead of adding TWT capability to scan results, we can display this information in connection status. Closing this PR.

**Pull request closed without merging**